frontend/docs/docs/user-guide/running-crawl.md

```diff
@@ -34,6 +34,17 @@ Like exclusions, the number of [browser windows](workflow-setup.md#browser-windo
 
 Unlike exclusions, this change will not be applied to future workflow runs.
 
+## Pausing and Resuming Crawls
+
+If you need to pause your crawl at any point after it has started, whether to reassess, rescope, or for any other reason, you now have the option to do so.
+
+Pausing a crawl will halt the timer and prevent any increase in the number of pages crawled or the overall size. The crawl status will change from Running to Pausing to eventually Paused to signify that the crawler is no longer crawling.
```

Comment:
Nit--when referring to a capitalized status like `Running` I would either bold it (`**Running**`) or wrap it in quotes (`“Running”`)